packages/react-magma-dom/src/components/TreeView/TreeView.stories.tsx

```diff
@@ -209,13 +209,22 @@ export const Complex = (args: Partial<TreeViewProps>) => {
   );
   const total = selectedItems?.length || 0;
 
+  const handleExpandedChange = (event :React.SyntheticEvent) => {
+    console.log(event);
+  }
```

Comment:
```suggestion
  const handleExpandedChange = (event: React.SyntheticEvent, items) => {
    console.log(event, items);
    setExpandedItems(items);
  };
```

Then under the button group we can add something like this to keep track of what items are being passed back
```
      <Spacer axis={SpacerAxis.vertical} size={24} />
      <p>Expanded: {expandedItems?.join(', ')}</p>
```

When I did this locally, I noticed that when `initialExpandedItems` is passed, the array will always return them even after they have been collapsed:

![image](https://github.com/user-attachments/assets/60284c17-37b1-4122-9e45-89511d05aed9)

Reply:
done